packages/pharos-site/src/components/statics/PageSection.tsx

```diff
@@ -141,6 +141,7 @@ const PageSection: FC<PageSectionProps> = ({
     );
     setDisplay(content);
   }, [
+    topMargin,
```

Comment:
This variable was missing from the React useEffect hook dependency and caused an error while upgrading. Manually added it to this PR.